manager/src/lib.rs

```diff
@@ -70,7 +71,7 @@ pub struct Task {
     pub gas: Gas,
 
     // NOTE: Only allow static pre-defined bytes
-    pub arguments: Vec<u8>,
+    pub arguments: Base64VecU8,
```

Comment:
Aha, so since we changed the Task here, the simulation test needed to have the hash changed. Was wondering how the hash was different. Pushed up with this commit: https://github.com/Cron-Near/contracts/pull/48/commits/65fd676844df7391821dc4bbcab9518d3abac932

Reply:
@mikedotexe actually - its more likely from me changing from keccak to sha

Reply:
ah yes